code/modules/uplink/uplink_items.dm

```diff
@@ -273,6 +274,26 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			U.purchase_log.LogPurchase(goods, I, 0)
 	return C
 
+#define RANDOM_KIT_TC_VALUES list(5, 6, 8, 10, 12, 14, 16, 18, 20, 25, 28, 32, 36, 40, 45, 50, 55, 60, 65, 100)	//Nat 20 used to be a nuke op creator so uhhh thats better at least
+
+//Will either give you complete crap or overpowered as fuck gear
+/datum/uplink_item/bundles_TC/surplus/random
+	name = "Syndicate Lootbox"
+	desc = "A dusty crate from the back of the Syndicate warehouse. Rumored to contain a valuable assortment of items, \
+			With their all new kit, codenamed 'scam' the syndicate attempted to extract the energy of the die of fate to \
+			make a loot-box style system but failed, so instead just fake their randomness using a corgi to sniff out the items to shove in it.\
+			Item price not guaranteed. Can contain normally unobtainable items."
+	check_include_modes = FALSE
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops, /datum/game_mode/incursion)
+	player_minimum = 30
+
+/datum/uplink_item/bundles_TC/surplus/random/purchase(mob/user, datum/component/uplink/U)
+	var/value_index = rand(1, 20)
```

Comment:
bro just roll it randomly between 5 and 100 instead of using the shittiest die to exist

Reply:
Ok I'll use some curve equation though so it's weighted towards shitter items

Reply:
There